src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,71 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    public static final int ALLOWABLE_RANGE_OF_UNITS = 50;
+    private ColorSupplier cs = new ColorSupplier();
```

Comment:
it would be better to give our variables more informative names (`cs` -> `colorSupplier`)